include/ttmlir/Dialect/TTIR/IR/TTIROps.td

```diff
@@ -288,6 +288,27 @@ def TTIR_NegOp: TTIR_ElementwiseUnaryOp<"neg"> {
     }];
 }
 
+def TTIR_LogitOp: TTIR_ElementwiseUnaryOp<"logit"> {
```

Comment:
LogitOp is an unfortunate name for this, as this is the logistic(sigmoid). The name logit is usually used for a different function, as can be seen in [torch.special.logit](https://pytorch.org/docs/stable/special.html#torch.special.logit) and [ttnn.logit](https://docs.tenstorrent.com/ttnn/latest/ttnn/ttnn/logit.html#ttnn.logit) (the doc page is sparse on details but from reading the ttnn [code](https://github.com/tenstorrent/tt-metal/blob/4584bc3bfd1d4ec8ad50222e40097d2c76c83ca8/ttnn/ttnn/operations/unary.py#L507) you can see it expects to behave the same as torch) . Additionally, in the meantime somebody else landed a ttir::SigmoidOp so this Op is no longer needed. I would suggest removing it, as logit is not a very common function in models

Reply:
I refactored with the changes so that stablehlo logistic op will convert to ttir.sigmoid op.